test/colony.js

```diff
@@ -123,30 +125,87 @@ contract("Colony", addresses => {
 
   describe("when working with permissions", () => {
     it("should be able to add a colony owner", async () => {
-      await authority.setUserRole(OTHER, 0, true);
-      const owner = await authority.hasUserRole.call(OTHER, 0);
+      await colonyAuthority.setUserRole(OTHER, 0, true);
+      const owner = await colonyAuthority.hasUserRole.call(OTHER, 0);
       assert.isTrue(owner);
     });
 
     it("should be able to add a colony admin", async () => {
-      await authority.setUserRole(OTHER, 1, true);
-      const admin = await authority.hasUserRole.call(OTHER, 1);
+      await colonyAuthority.setUserRole(OTHER, 1, true);
+      const admin = await colonyAuthority.hasUserRole.call(OTHER, 1);
       assert.isTrue(admin);
     });
 
     it("should be able to remove a colony owner", async () => {
-      await authority.setUserRole(OTHER, 0, true);
-      await authority.setUserRole(OTHER, 0, false);
-      const owner = await authority.hasUserRole.call(OTHER, 0);
+      await colonyAuthority.setUserRole(OTHER, 0, true);
+      await colonyAuthority.setUserRole(OTHER, 0, false);
+      const owner = await colonyAuthority.hasUserRole.call(OTHER, 0);
       assert.isFalse(owner);
     });
 
     it("should be able to remove a colony admin", async () => {
-      await authority.setUserRole(OTHER, 1, true);
-      await authority.setUserRole(OTHER, 1, false);
-      const admin = await authority.hasUserRole.call(OTHER, 1);
+      await colonyAuthority.setUserRole(OTHER, 1, true);
+      await colonyAuthority.setUserRole(OTHER, 1, false);
+      const admin = await colonyAuthority.hasUserRole.call(OTHER, 1);
       assert.isFalse(admin);
     });
+
+    it("should allow admin to assign admin role", async () => {
+      const adminRole = 1;
+
+      const address1 = addresses[1];
+      const address2 = addresses[5];
```

Comment:
Since an `address` can be a contract or a user, can we call these `user1` and `user5` or something like that?